assets/docs/vasudevgupta7/models/wav2vec2-base/1.md

```diff
@@ -0,0 +1,37 @@
+# Module vasudevgupta7/wav2vec2-base/1
+
+Pre-trained speech model from Facebook for Automatic Speech Recognition
+
+<!-- asset-path:  https://huggingface.co/vasudevgupta/tf-wav2vec2-base/resolve/main/wav2vec2-base.tar.gz -->
+<!-- task: audio-stt -->
+<!-- network-architecture: wav2vec2-base -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: true -->
+<!-- license: apache-2.0 -->
+<!-- language: en -->
+<!-- colab: https://colab.research.google.com/github/vasudevgupta7/gsoc-wav2vec2/blob/main/notebooks/wav2vec2_base_saved_model.ipynb -->
+
+## Overview
+
+This model is TensorFlow equivalent of [`facebook/wav2vec2-base`](https://huggingface.co/facebook/wav2vec2-base).
+
+**How to use this model?**
+
+```python
+import tensorflow as tf
+import tensorflow_hub as hub
+
+loaded = hub.load("https://tfhub.dev/vasudevgupta7/wav2vec2-base/1")
+
+# Load this model for training
+model = hub.KerasLayer(loaded.signatures["train"], trainable=True)
+# Now this model can be used for training like any other TF saved-model
+
+# How to model for inference?
+# just change signature to `infer`
+model = hub.KerasLayer(loaded.signatures["infer"], trainable=False)
+
+@tf.function(jit_compile=True)
+def forward(speech):
+    return model(speech)
+```
```

Comment:
Would suggest including two things:

* Distinction between the notebooks that show inference and fine-tuning workflows. 
* A reference to the paper that implements this model.

Reply:
This model shouldn't be directly used for inference as it's not having any head over the pre-trained model. So, removing the direct inference thing. Added workflow to add LM head & fine-tune the whole model.